rdr_service/alembic/versions/16452fdb1334_summary_cope_survey_fields.py

```diff
@@ -0,0 +1,79 @@
+"""summary cope survey fields
+
+Revision ID: 16452fdb1334
+Revises: c0394a487b8b
+Create Date: 2020-05-12 14:02:39.592019
+
+"""
+from alembic import op
+import sqlalchemy as sa
+import model.utils
+from sqlalchemy.dialects import mysql
+
+from rdr_service.participant_enums import PhysicalMeasurementsStatus, QuestionnaireStatus, OrderStatus
+from rdr_service.participant_enums import WithdrawalStatus, WithdrawalReason, SuspensionStatus, QuestionnaireDefinitionStatus
+from rdr_service.participant_enums import EnrollmentStatus, Race, SampleStatus, OrganizationType, BiobankOrderStatus
+from rdr_service.participant_enums import OrderShipmentTrackingStatus, OrderShipmentStatus
+from rdr_service.participant_enums import MetricSetType, MetricsKey, GenderIdentity
+from rdr_service.model.base import add_table_history_table, drop_table_history_table
+from rdr_service.model.code import CodeType
+from rdr_service.model.site_enums import SiteStatus, EnrollingStatus, DigitalSchedulingStatus, ObsoleteStatus
+
+# revision identifiers, used by Alembic.
+revision = '16452fdb1334'
+down_revision = 'c0394a487b8b'
+branch_labels = None
+depends_on = None
+
+
+def upgrade(engine_name):
+    globals()["upgrade_%s" % engine_name]()
+
+
+def downgrade(engine_name):
+    globals()["downgrade_%s" % engine_name]()
+
+
+
+def upgrade_rdr():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column('participant_summary', sa.Column('questionnaire_on_cope_july', rdr_service.model.utils.Enum(QuestionnaireStatus), nullable=True))
+    op.add_column('participant_summary', sa.Column('questionnaire_on_cope_july_authored', rdr_service.model.utils.UTCDateTime(), nullable=True))
+    op.add_column('participant_summary', sa.Column('questionnaire_on_cope_july_time', rdr_service.model.utils.UTCDateTime(), nullable=True))
+    op.add_column('participant_summary', sa.Column('questionnaire_on_cope_june', rdr_service.model.utils.Enum(QuestionnaireStatus), nullable=True))
+    op.add_column('participant_summary', sa.Column('questionnaire_on_cope_june_authored', rdr_service.model.utils.UTCDateTime(), nullable=True))
+    op.add_column('participant_summary', sa.Column('questionnaire_on_cope_june_time', rdr_service.model.utils.UTCDateTime(), nullable=True))
+    op.add_column('participant_summary', sa.Column('questionnaire_on_cope_may', rdr_service.model.utils.Enum(QuestionnaireStatus), nullable=True))
+    op.add_column('participant_summary', sa.Column('questionnaire_on_cope_may_authored', rdr_service.model.utils.UTCDateTime(), nullable=True))
+    op.add_column('participant_summary', sa.Column('questionnaire_on_cope_may_time', rdr_service.model.utils.UTCDateTime(), nullable=True))
+    # ### end Alembic commands ###
+
+
+def downgrade_rdr():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.alter_column('resource_search_results', 'page_no',
```

Comment:
I don't see the inverse of this statement in `upgrade_rdr()`. Is this supposed to be here?

Reply:
Nope. I'm not sure where it came from when I generated the migration, but I'll remove it.